base/uk.ac.stfc.isis.ibex.ui.graphing/src/uk/ac/stfc/isis/ibex/ui/graphing/websocketview/MatplotlibFigureViewModel.java

```diff
@@ -284,4 +354,23 @@ public void setCursorPosition(final MatplotlibCursorPosition cursorPosition) {
 		this.cursorPosition = cursorPosition;
 		cursorPositionChanged.set(true);
 	}
+	
+	/**
+	 * Notifies the websocket model that a mouse button as been pressed/released over the figure 
+	 * @param pressType
+	 */
+	public void notifyButtonPressed(String pressType) {
```

Comment:
Would `pressType` be better as an Enum so that we can be sure we only ever pass valid values?